hcfb-be/backend-server/app/model/distribution.model.js

```diff
@@ -1,24 +1,24 @@
-module.exports = (sequelize, Sequilize) => {
-    const Distribution = sequilize.define('distribution', {
+module.exports = (sequelize, Sequelize) => {
+    const Distribution = sequelize.define('distribution', {
         deliveryLocation: {
             type: Sequelize.STRING
         },
-        deliveryTime: {
+        deliveryDepartureTime: {
             type: Sequelize.DATE
         },
         deliveryWeights : {
-            type: Sequelize.ARRAY(Sequilize.TEXT),
+            type: Sequelize.ARRAY(Sequelize.DOUBLE),
             allowNull: true,
             validate: {
                 isSpecificLength(value) {
-                    if(value.length !== 6) {
-                        throw new Error("deliveryWeights array must have exactly 6 items.")
+                    if(value.length != 7) {
```

Comment:
```suggestion
                    if(value.length !== 7) {
```

Triple-unequals to avoid the type coercion here.